sql/modules/Menu.sql

```diff
@@ -102,6 +112,18 @@ It is thus similar to menu_generate() but it only returns the menu items
 associated with nodes directly descendant from the parent.  It is used for
 menues for frameless browsers.$$;
 
+CREATE OR REPLACE FUNCTION menu_item(in_id int) RETURNS SETOF menu_item
```

Comment:
Do we really need this? I mean, we've sent the menu to the client already, haven't we? Can't the client just use what we've sent it?